<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MediaInterval_Widget</class>
 <widget class="QWidget" name="MediaInterval_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Data Type:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Digital Interval</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Data Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="name_label">
       <property name="text">
        <string>Name</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="ColorPicker_Widget" name="color_picker" native="true"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="plotting_style_layout">
     <item>
      <widget class="QLabel" name="label_plotting_style">
       <property name="text">
        <string>Plotting Style:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="plotting_style_combobox">
       <property name="toolTip">
        <string>Select the visualization style for interval indicators</string>
       </property>
       <item>
        <property name="text">
         <string>Box</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Border</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="style_options_stack">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="box_style_page">
      <layout class="QVBoxLayout" name="box_style_layout">
       <item>
        <widget class="BoxIntervalStyle_Widget" name="box_style_widget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="border_style_page">
      <layout class="QVBoxLayout" name="border_style_layout">
       <item>
        <widget class="BorderIntervalStyle_Widget" name="border_style_widget" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorPicker_Widget</class>
   <extends>QWidget</extends>
   <header>ColorPicker_Widget/ColorPicker_Widget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BoxIntervalStyle_Widget</class>
   <extends>QWidget</extends>
   <header>Media_Widget/UI/SubWidgets/MediaInterval_Widget/StyleWidgets/BoxIntervalStyle_Widget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BorderIntervalStyle_Widget</class>
   <extends>QWidget</extends>
   <header>Media_Widget/UI/SubWidgets/MediaInterval_Widget/StyleWidgets/BorderIntervalStyle_Widget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
